src/main/java/taxi/model/Driver.java

```diff
@@ -50,11 +74,24 @@ public boolean equals(Object o) {
         Driver driver = (Driver) o;
         return Objects.equals(id, driver.id)
                 && Objects.equals(name, driver.name)
-                && Objects.equals(licenseNumber, driver.licenseNumber);
+                && Objects.equals(licenseNumber, driver.licenseNumber)
+                && Objects.equals(login, driver.login)
+                && Objects.equals(password, driver.password);
```

Comment:
```suggestion
                
```
Do we really need this field here? If the driver changed his password we will recieve another driver, but I think this is the same driver.